backend/api/views.py

```diff
@@ -258,7 +265,8 @@ class PhotoViewSet(ModelViewSet):
             400: ErrorResponseSerializer,
             401: ErrorResponseSerializer,
             403: ErrorResponseSerializer,
-        }
+        },
+        auth=[]
```

Comment:
Тут тоже, соответственно:
```suggestion
        }
```